modules/olm-catalogsource-image-template.adoc

```diff
@@ -18,14 +18,14 @@ During a cluster upgrade, the index image tag for the default Red Hat-provided c
 
 [source,terminal]
 ----
-registry.redhat.io/redhat/redhat-operator-index:v4.19
+registry.redhat.io/redhat/redhat-operator-index:v4.20
```

Comment:
just calling this and line 28 out because they're bumping the OCP version instead of the kube version. Did you mean for these changes to be included in this PR?